utils/timezone.jsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import UserStore from 'stores/user_store.jsx';
+import * as UserActions from 'actions/user_actions.jsx';
+
+const dateTimeFormat = new Intl.DateTimeFormat();
+
+export function getSupportedTimezones() {
+    if (global.mm_config && global.mm_config.SupportedTimezones) {
+        return global.mm_config.SupportedTimezones.split(',');
+    }
+    return [];
+}
+
+export function getBrowserTimezone() {
+    return dateTimeFormat.resolvedOptions().timeZone;
```

Comment:
[Mozilla developer docs for `Intl.DateTimeFormat.prototype.resolvedOptions().timeZone`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/resolvedOptions) says there's no support in IE Edge and some mobile browsers. I just want to verify that support for this feature is sufficient and/or we have a fallback.